### DISCUSSION TITLE
Better Gold Pickup Sound Effect

### DISCUSSION THREAD

**Discussion by Deadstarrr** [diasurgical/DevilutionX#6900] [Ideas]
Would it be possible to change the auto pickup sound to the same sound effect as the gold drop sound effect? Currently the pickup sound is a generic sound that is the same for all items, but it would be much cooler if the gold drop sound effect was played when you picked up gold specifically.

**Comment by Trihedraf** [comment]
If it made the same sound as a drop how would you know that it was picked up vs another gold pile dropped elsewhere

**Comment by julealgon** [comment]
Why wouldn't you then want to apply this same logic for every single item? Singling out gold on this makes zero sense to me personally.

As this is proposed, I don't agree it is a good change.
